core/common/src/main/java/alluxio/security/authorization/DefaultAccessControlList.java

```diff
@@ -189,18 +187,18 @@ public boolean equals(Object o) {
       return false;
     }
     DefaultAccessControlList that = (DefaultAccessControlList) o;
+    boolean extendedEquals = (mExtendedEntries == null && that.mExtendedEntries == null)
```

Comment:
should this be changed to consider `null` and empty `extendedEntries` the same?  like `AccessControlList`?

Reply:
I updated this (like acl).